string traj_name
---
uint16[] traj_id